allennlp/commands/__init__.py

```diff
@@ -51,37 +50,33 @@ def add_argument(self, *args, **kwargs):
 
 
 def create_parser(
-    prog: str = None, subcommand_overrides: Dict[str, Subcommand] = None
+    prog: str = None, subcommand_overrides: Iterable[Subcommand] = None
 ) -> argparse.ArgumentParser:
     """
     Creates the argument parser for the main program.
     """
-    if subcommand_overrides is None:
-        subcommand_overrides = {}
+    subcommand_overrides = subcommand_overrides or []
 
-    parser = ArgumentParserWithDefaults(description="Run AllenNLP", usage="%(prog)s", prog=prog)
```

Comment:
I removed `usage` because just the output "usage: allennlp\n" doesn't make sense to me.